pennylane/measure.py

```diff
@@ -44,15 +44,21 @@ class MeasurementProcess:
 
     # pylint: disable=too-few-public-methods
 
-    def __init__(self, return_type, obs=None, wires=None, eigvals=None):
+    def __init__(self, return_type, obs=None, **kwargs):
+        eigvals = kwargs.get("eigvals", None)
+        wires = kwargs.get("wires", None)
```

Comment:
Codefactor emitted an issue: the signature had too many arguments by adding `shape` as a new one too. I've readded `obs` as an `obs=None` parameter because ~25 test cases passed the observable as a positional argument.

Reply:
oh this is definitely fine to override! I think we override in a lot of places :)